template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+        </ul>
+    </td>
+    <td><ul>
+          <li>The Good Docs Project welcoming committee</li>
+          <li>Working groups</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>2 - Adopt a template</td>
+    <td><ul>
+          <li>Join a template writing working group led by a template mentor.</li>
+          <li>Work with a template mentor and your working group to decide which template you’ll work on.</li>
+          <li>Assign yourself to the corresponding issue for that template.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>The issues board on the templates repo</li>
+          <li>Template mentors</li>
+          <li>Templates coordinator</li>
+          <li>Template working groups</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>3 - Research and draft</td>
+    <td><ul>
+          <li>Research examples and best practices for the type of template you are working on.</li>
+          <li>Collaborate and get early feedback on your research from your template mentor and/or other templateers as part of a template writing working group.</li>
+          <li>Create drafts of the template set deliverables in Google Docs or your preferred tool.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>Working groups</li>
+          <li>Template mentors</li>
+          <li>Templates coordinator</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>4 - Workshop the drafts with the community</td>
+    <td><ul>
+          <li>Revise and refine the drafts of your deliverables with help from at least three community reviewers.</li>
+          <li>Optional: Revise and refine your draft with individuals beyond our community (such as Write the Docs or subject matter experts).</li>
+          <li>Work with your template mentor and the templates coordinator to determine when your draft is ready for the next phase.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>Community reviewers</li>
+          <li>Subject matter experts and members of other technical writing communities</li>
+          <li>Template mentors</li>
+          <li>Templates coordinator</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>5 - Submit a pull request</td>
+    <td><ul>
+          <li>If needed, convert drafts from Google Docs or your preferred tool to Markdown.</li>
+          <li>Open a pull request. NOTE: If you are unfamiliar with Git and GitHub, your template mentor or another community member can help you.</li>
+          <li>Revise documents based on requests from pull request reviewers.</li>
+        </ul>
+    </td>
+    <td><ul>
+          <li>Pull request reviewers</li>
+          <li>Template mentors</li>
+          <li>Templates coordinator</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>6 - Improve the template with user feedback</td>
+    <td><ul>
+          <li>After completing the previous phase, your template will officially be part of the Good Docs Project and will be available to our users.</li>

```

Comment:
We might want to add a requirement here to create an issue to handle feedback and do the next version of the template, after it is finalized. This can then be assigned directly to the creator, or left unassigned (surfacing clearly that there is a gap in coverage, to facilitate governance of the content). But perhaps the template coordinator or mentors could do that.

Reply:
Let's leave this as an open action item that needs to be resolved when we work out the finer details of the template project management process.

Reply:
I've added your comment to the feedback section of the RFC.